diff --git a/kded/CMakeLists.txt b/kded/CMakeLists.txt
index 8c0d71a..418577b 100644
--- a/kded/CMakeLists.txt
+++ b/kded/CMakeLists.txt
@@ -71,6 +71,7 @@ install(TARGETS kbuildsycoca4 ${INSTALL_TARGETS_DEFAULT_ARGS})
 install(
     FILES applications.menu
     DESTINATION ${KDE4_SYSCONF_INSTALL_DIR}/xdg/menus
+    RENAME kde-applications.menu
 )
 
 install(
diff --git a/kded/kbuildsycoca.cpp b/kded/kbuildsycoca.cpp
index 1b2c2cc..9acfc3f 100644
--- a/kded/kbuildsycoca.cpp
+++ b/kded/kbuildsycoca.cpp
@@ -304,7 +304,7 @@ bool KBuildSycoca::build()
      if (!m_trackId.isEmpty())
         g_vfolder->setTrackId(m_trackId);
 
-     VFolderMenu::SubMenu *kdeMenu = g_vfolder->parseMenu("applications.menu");
+     VFolderMenu::SubMenu *kdeMenu = g_vfolder->parseMenu("kde-applications.menu");
 
      KServiceGroup::Ptr entry = g_buildServiceGroupFactory->addNew("/", kdeMenu->directoryFile, KServiceGroup::Ptr(), false);
      entry->setLayoutInfo(kdeMenu->layoutList);
